.github/workflows/build-and-publish-next.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish LFNext images to Docker Hub on-demand
+
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on
+on:
+  workflow_dispatch:
+
+jobs:
+  build-n-publish:
+    runs-on: ubuntu-latest
+
+    env:
+      # https://docs.docker.com/develop/develop-images/build_enhancements/
+      DOCKER_BUILDKIT: 1
+
+    defaults:
+      run:
+        working-directory: docker
+
+    steps:
+      -
+        uses: actions/checkout@v2
+      -
+        run: make build-next
```

Comment:
yeah, I'm definitely open to this idea... I'd like to simplify things if possible.

Reply:
Resolving this conversation so that we won't be blocked from merging the PR. If you have more to say, @longrunningprocess, I'd be happy to carry on the conversation(s) in regular comments.